package.json

```diff
@@ -38,15 +36,19 @@
     "@babel/plugin-proposal-class-properties": "^7.16.7",
     "@babel/plugin-transform-modules-commonjs": "^7.16.8",
     "@babel/preset-env": "^7.16.11",
+    "antlr4": "^4.9.2",
```

Comment:
When we install "^4.9.2" of antlr, we end up with version 4.11.0
When I tried this, the antlr-based code generation / runtime failed -- they changed the way the modules are built in later versions of antlr.
Did you try doing a fresh `npm install` and code generation?

Reply:
that change was unintentional. Reverted it.